mesonbuild/interpreter/interpreter.py

```diff
@@ -3186,7 +3188,7 @@ def add_target(self, name: str, tobj: build.Target) -> None:
             self.coredata.target_guids[idname] = str(uuid.uuid4()).upper()
 
     @FeatureNew('both_libraries', '0.46.0')
-    def build_both_libraries(self, node, args, kwargs):
+    def build_both_libraries(self, node, args, kwargs: T.Dict[str, T.Any]):
```

Comment:
TYPE_kwargs?

Reply:
We can actually do better pretty trivially. Next revision has actual correct annotations added in a separate patch